torch/custom_class.h

```diff
@@ -163,40 +170,15 @@ class class_ {
  private:
   template <typename Func>
   void defineMethod(std::string name, Func func) {
-    auto graph = std::make_shared<Graph>();
     auto qualFuncName = className + "::" + name;
-    ensure_c10_registerer_defined();
     registeredOps().push_back(
         torch::RegisterOperators().op(qualFuncName, std::move(func)));
     auto func_symbol = c10::Symbol::fromQualString(qualFuncName);
-    auto ops = torch::jit::getAllOperatorsFor(func_symbol);
-    TORCH_CHECK(ops.size() == 1);
-    auto &schema = ops[0]->schema();
-
-    for (const auto& arg : schema.arguments()) {
-      graph->addInput()->setType(arg.type());
-    }
-
-    auto opCall = graph->insertNode(graph->create(
-        func_symbol, graph->inputs(), schema.returns().size()));
-    Value* res;
-    if (schema.returns().size() > 1) {
-      const auto& returns = schema.returns();
-      size_t op_invocation_idx = 0;
-      for (const auto& ret : returns) {
-        opCall->output(op_invocation_idx++)->setType(ret.type());
-      }
-      res = graph->insertNode(graph->createTuple(opCall->outputs()))->output();
-    } else if (schema.returns().size() == 1) {
-      const auto& returns = schema.returns();
-      res = opCall->output()->setType(returns[0].type());
-    } else {
-      res = graph->insertConstant(IValue())->setType(NoneType::get());
-    }
-    graph->registerOutput(res);
-
-    auto method = classCU()->create_function(qualClassName + "." + name, graph);
-    classTypePtr->addMethod(method);
+
```

Comment:
Rather than register an operator here, use the boxing facilities of c10 to generate a `std::function<void(Stack&)>` and then use that to directly construct a BuiltinOpFunction.